### PR TITLE
[Fixes #183] GeoNode User Guide

### LINKED ISSUE #183: GeoNode User Guide
https://docs.geonode.org/en/master/usage/index.html#geonode-users-guide

### PR DESCRIPTION
**Pull request by marthamareal**

### CODE REVIEW

**Review by giohappy — CHANGES_REQUESTED** [review]
@marthamareal regarding permissions, it would be good to have an explanation of what the different grants (View, Download, Edit, Manager) offer for each type of resource.

**Comment by marthamareal** [comment]
updated PR with the changes

File: usage/managing_datasets/dataset_permissions.rst [hunk 1]
```diff
@@ -1,31 +1,31 @@
 .. _dataset-permissions:
 
-Changing the Dataset Permissions
-==============================
+Share OPtions
```
Comment (gannebamm) [inline usage/managing_datasets/dataset_permissions.rst:3]: typo `Options`

**Review by fernandinand — APPROVED** [review]
LGTM. Just confirm if thumbnail image needs update

File: usage/managing_documents/document_editing.rst [hunk 2]
```diff
@@ -3,35 +3,19 @@
...
 
 Setting the Document Thumbnail
 ------------------------------
 
-From the *Editing Panel*, it is also possible to *Set the Thumbnail* of the document. Click on :guilabel:`Set` to open the *Thumbnail Uploading* page and chose the image that will illustrate your document. You can either drag and drop it in the *Drop files here* box or selecting from your folders by clicking on *Choose Files*. Once this is done, click on the red button :guilabel:`Upload files`. If the thumbnail has been successfully uploaded you can see it by coming back to the document list. Click on the :guilabel:`Explore Documents` button to check that.
+From the *Metadata Form*, it is possible to *Set the Thumbnail* of the document. Click on :guilabel:`Choose file` to open the *Thumbnail Uploading* page and chose the image that will illustrate your document. Once this is done, click on the ok button and save. If the thumbnail has been successfully uploaded you can see it by coming back to the document list.
 
 .. figure:: img/document_uploading_thumbnail.png
```
[the remark this answers is not in the document]
Comment (marthamareal) [reply usage/managing_documents/document_editing.rst:13]: @fernandinand I used a black background image, do you suggest using another visible image?
doc

[the remark this answers is not in the document]
Comment (fernandinand) [reply usage/managing_documents/document_editing.rst:13]: @marthamareal Apologies, I thought this was an error on image upload/screenshot. Although is not the best image we can leave it for now.

[the remark this answers is not in the document]
Comment (giohappy) [reply usage/managing_documents/document_editing.rst:13]: @marthamareal @fernandinand I guess it won't take much recreating a better screenshot,..

**Pull request merged**